provider/core/src/marker.rs

```diff
@@ -10,6 +10,8 @@ use crate::yoke::Yokeable;
 /// Trait marker for data structs. All types delivered by the data provider must be associated with
 /// something implementing this trait.
 ///
+/// Structs implementing this trait are normally generated with the [`data_struct!`] macro.
```

Comment:
Not a function-like macro.

```suggestion
/// Structs implementing this trait are normally generated with the [`#[data_struct]`] macro.
```

Reply:
that linkification may not work, try `[....](macro@data_struct)`